PukingPredator/Assets/Scripts/Eating.cs

```diff
@@ -33,6 +33,9 @@ public class Eating : InputBehaviour
     // Time when the right mouse button was pressed
     private float rightMouseButtonDownTime;
     private float pukeForce;
+    private const float MINPUKEFORCE = 10f;

```

Comment:
wrong naming format for const, put underscores between words to make it readable if youre doing a const